crates/crypto-utils-evm/src/lib.rs

```diff
@@ -0,0 +1,160 @@
+//! Various crypto helper functions for EVM.
+
+use sp_core::{H160, H256};
+
+/// A structure representing the key information.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct KeyData {
+    /// The account address.
+    pub account: H160,
+    /// The private key of this account.
+    pub private_key: H256,
+    /// The mnemonic phrase.
+    pub mnemonic: String,
+    /// The drivation path used for this account.
+    pub derivation_path: bip32::DerivationPath,
+}
+
+impl KeyData {
+    /// Create a new [`KeyData`] from the given BIP39 mnemonic and an account index.
+    pub fn from_mnemonic_bip39(
+        mnemonic: &bip39::Mnemonic,
+        password: &str,
+        derivation_path: &bip32::DerivationPath,
+    ) -> Self {
+        // Retrieve the seed from the mnemonic.
+        let seed = bip39::Seed::new(mnemonic, password);
+
+        // Derives the private key from.
+        let ext = bip32::XPrv::derive_from_path(seed, derivation_path).unwrap();
```

Comment:
Are we going just doing `unwrap` without returning any error ?

Reply:
Not really, we need to do something about it; also, a crate and unused vars cleanup is pending.

Reply:
Should be ready now, please take another look

Reply:
One `unwrap` was missed. Everything is fine. Thanks.